lib/internal/worker.js

```diff
@@ -261,10 +264,24 @@ class Worker extends EventEmitter {
     // Actually start the new thread now that everything is in place.
     this[kHandle].startThread();
 
+    if (~~options.timeout > 0) {
```

Comment:
Why the ~~s instead of explicit conversion? Especially in a case where coercion is happening anyway  for the `>`? (similarly in the setTimeout - and also I think this is missing validation

Reply:
Thanks ! fixed.